portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,20 +15,24 @@
     
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> comments = new ArrayList<String>();
-    comments.add("Black Lives Matter!"); 
-    comments.add("The beginnings of a nice website, Jaz!");
-    comments.add("Website still needs some work but it's coming together! Must be learning some HTML/CSS."); 
-    String json = convertToJsonUsingGson(comments);    
-    
+    ArrayList<String> comments = commentGenerator();
+    String json = toJsonString(comments);    
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
-  private String convertToJsonUsingGson(ArrayList<String> comments){
+  private String toJsonString(ArrayList<String> comments){
       Gson gson = new Gson();
       String json = gson.toJson(comments);
       return json;
 
   }
+
+  public ArrayList<String> commentGenerator(){
```

Comment:
commentGenerator is not a good name. it doesnt describe what this function is or its behavior. Rename to buildFakeComments() or getFakeComments(). Try to use standard programming names "getBlah()" or "buildBlah()" to minimize how much people need to dig into ur code